avalonia-extensions.sln.DotSettings

```diff
@@ -51,6 +51,11 @@
   &amp;lt;option name="myName" value="StyleCop" /&amp;gt;
 &amp;lt;/profile&amp;gt;&lt;/IDEA_SETTINGS&gt;&lt;CSArrangeQualifiers&gt;True&lt;/CSArrangeQualifiers&gt;&lt;CSFixBuiltinTypeReferences&gt;True&lt;/CSFixBuiltinTypeReferences&gt;&lt;/Profile&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/CodeCleanup/SilentCleanupProfile/@EntryValue">Built-in: Reformat &amp; Apply Syntax Style</s:String>
+	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpCodeStyle/BRACES_FOR_FOR/@EntryValue">Required</s:String>
+	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpCodeStyle/BRACES_FOR_FOREACH/@EntryValue">Required</s:String>
+	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpCodeStyle/BRACES_FOR_IFELSE/@EntryValue">RequiredForMultilineStatement</s:String>
+	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpCodeStyle/BRACES_FOR_WHILE/@EntryValue">Required</s:String>
+	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpCodeStyle/BRACES_REDUNDANT/@EntryValue">False</s:Boolean>
```

Comment:
Hm, something is still not right with our project-wide rules ... this PR seems to have lots of formatting changes not related to the rules you added here (indents, var, single-line if statements ... (doesn't need to be resolved in this PR, but I'd love to get my head around it)

Reply:
Yeah, I fixed that because my previous rule was removing brackets on foreachs that had only one instruction, which is not something we want.
Let me know what other formatting  you think is wrong? I'm not quite sure

Reply:
Well looking at the diff, it has changed the indents back to three in lots of places (e.g. src/Devolutions.AvaloniaTheme.MacOS/Devolutions.AvaloniaTheme.MacOS.csproj) and it has changed things like `Window? oldWindow = desktopLifetime.MainWindow;` back to `var oldWindow = desktopLifetime.MainWindow;` in various places ... I actually started looking into resharper in response to this post https://devolutions.slack.com/archives/C07SZMYG2CA/p1744987765580969 ... where he said to avoid vars

Reply:
It should all be reverted now

Reply:
Looks like we're back to 4 indents for C# ? Whatever it is, is fine by me - as long as we've got the rules nailed down to avoid formatting to changes depending on who last edited (We can run a format-all in a separate PR with whatever we set as final rules)